apps/extension/public/manifest.json

```diff
@@ -8,6 +8,10 @@
     "default_title": "Talisman",
     "default_popup": "popup.html?embedded#/portfolio"
   },
+  "options_ui": {
```

Comment:
Adds this button to the toolbar:

![Screenshot 2023-11-30 at 10 39 00](https://github.com/TalismanSociety/talisman/assets/2741569/db135b5f-1fcd-4e58-825e-48fe05ae11ac)